homework/index.js

```diff
@@ -0,0 +1,21 @@
+const repoSelectElement = document.getElementById('repositories');
+const unContainer = document.querySelector('.right__div-username');
```

Comment:
Note: here you use 2 underscores; in your html you only used one - so this won't select anything.